config/locales/en.yml

```diff
@@ -1232,187 +1232,187 @@ en:
-          miniature: "Miniature Rail"
+          junction: "Railroad Junction"
+          level_crossing: "Railroad Crossing"
+          light_rail: "Light Rail Track"
+          miniature: "Miniature Train Track"
           monorail: "Monorail"
-          narrow_gauge: "Narrow Gauge Railway"
-          platform: "Railway Platform"
-          preserved: "Preserved Railway"
-          proposed: "Proposed Railway"
-          rail: "Rail"
-          spur: "Railway Spur"
-          station: "Railway Station"
-          stop: "Railway Stop"
-          subway: "Subway"
+          narrow_gauge: "Narrow Gauge Track"
+          platform: "Train Platform"
+          preserved: "Preserved Train Track"
+          proposed: "Proposed Train Track"
+          rail: "Train Track"
+          spur: "Spur Track"
+          station: "Train Station"
+          stop: "Train Stopping Position"
+          subway: "Subway Track"
           subway_entrance: "Subway Entrance"
-          switch: "Railway Points"
-          tram: "Tramway"
-          tram_stop: "Tram Stop"
+          switch: "Railroad Switch"
+          tram: "Trolley Track"
+          tram_stop: "Trolley Stopping Position"
           turntable: "Turntable"
-          yard: "Railway Yard"
+          yard: "Rail Yard"
         shop:
-          agrarian: "Agrarian Shop"
-          alcohol: "Off License"
-          antiques: "Antiques"
-          appliance: "Appliances Shop"
-          art: "Art Shop"
-          baby_goods: "Baby Goods"
-          bag: "Bag Shop"
+          agrarian: "Farm Supply Shop"
+          alcohol: "Liquor Store"
+          antiques: "Antique Store"
+          appliance: "Appliances Store"
+          art: "Art Store"
+          baby_goods: "Baby Goods Store"
+          bag: "Bag Store"
           bakery: "Bakery"
-          bathroom_furnishing: "Bathroom Furnishing"
-          beauty: "Beauty Shop"
-          bed: "Bedding Products"
-          beverages: "Beverages Shop"
+          bathroom_furnishing: "Bathroom Furnishing Store"
+          beauty: "Beautician"
+          bed: "Bedding Store"
+          beverages: "Beverage Store"
           bicycle: "Bicycle Shop"
           bookmaker: "Bookmaker"
-          books: "Book Shop"
+          books: "Bookstore"
           boutique: "Boutique"
           butcher: "Butcher"
-          car: "Car Shop"
-          car_parts: "Car Parts"
-          car_repair: "Car Repair"
-          carpet: "Carpet Shop"
-          charity: "Charity Shop"
-          cheese: "Cheese Shop"
-          chemist: "Chemist"
-          chocolate: "Chocolate"
-          clothes: "Clothes Shop"
-          coffee: "Coffee Shop"
-          computer: "Computer Shop"
-          confectionery: "Confectionery Shop"
+          car: "Car Dealership"
+          car_parts: "Car Parts Store"
+          car_repair: "Car Repair Shop"
+          carpet: "Carpet Store"
+          charity: "Charity Store"
+          cheese: "Cheese Store"
+          chemist: "Drugstore"
```

Comment:
This is a notorious string among the U.S. mapping community. It’s quite difficult to fathom why a pharmacist would set up shop inside a chemistry lab. On the other hand, I hope Commonwealth English speakers who wind up seeing the replacement won’t think it refers to a vendor of illicit drugs. Personally, I haven’t been a fan of equating drugstores with `shop=chemist` (openstreetmap/iD#3213), but that’s a done deal now.